net/dcwifi/mrmctl/patches/01_enable_shared_libs.patch

```diff
@@ -0,0 +1,11 @@
+--- a/userland/configure.ac
```

Comment:
I tested without this patch. Seems to work just fine. AC_ENABLE_SHARED is overriden with the CONFIGURE_ARG AFAIK. You can also remove PKG_FIXUP as it is no longer needed.